api.planx.uk/modules/analytics/metabase/dashboard/types.ts

```diff
@@ -0,0 +1,94 @@
+import type { ValidatedRequestHandler } from "../../../../shared/middleware/validate.js";
+import type { ApiResponse } from "../shared/types.js";
+import { z } from "zod";
+
+export interface CreateNewDashboardParams {
```

Comment:
Is the interface just a copy of the Metabase type? It looks like there's a number of fields we're exposing but just not using (e.g. `name`, `collectionPosition`).

Whilst it's nice for completeness, it's probably simpler just to have a slimmed down version which contains the parameters we care about and actually want to set. It looks like these are already defined via the `Pick<CreateNewDashboardParams>` used for `CopyDashboardParams` and `UpdateFilterParams`. There's also no overlap between these two `Pick<CreateNewDashboardParams>`s so it may be simpler to define them separately.